src/ui/components/search/autocompletecomponent.js

```diff
@@ -214,7 +214,7 @@ export default class AutoCompleteComponent extends Component {
       resultIndex: this._resultIndex,
       promptHeader: this._originalQuery.length === 0 ? this.promptHeader : null,
       listLabelIdName: this.listLabelIdName,
-      autocompleteContainerIdName: `yxt-AutoComplete-container-${this._config.name.replaceAll('.', '-')}`,
+      autocompleteContainerIdName: `yxt-AutoComplete-container-${this._config.name ? this._config.name.replaceAll('.', '-') : ''}`,
```

Comment:
does `yxt-AutoComplete-container-${this._config.name?.replaceAll('.', '-')}` work instead?